### DISCUSSION TITLE
How to get product images

### DISCUSSION THREAD

**Discussion by serwin35** [lucasgiovanny/laravel-prestashop#1] [Q&A] [answered]
Hi, can You tell me, how I can get images to product ?

**Comment by lucasgiovanny** [comment, marked as answer]
I was just checking the Prestashop API documentation to get images and I'm going to need to work on an update to allow that. Right, now, it's not possible. I'll work on that soon.

**Reply by serwin35** [reply]
Ok Thx bro
